internal/network/command_access_point_private_link_egress_endpoint_list.go

```diff
@@ -0,0 +1,63 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *accessPointCommand) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List egress endpoints.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
```

Comment:
There are a few parameters for `List`, not sure if the bug in the SDK for `MultipleSearchFilter` is fixed...

Reply:
I only saw `display_name` for access point list in the SDK. Are there others?

Reply:
That's the only one, I copy pasta my comment. Sorry for the confusion.